rust/hyperlane-core/Cargo.toml

```diff
@@ -17,7 +17,6 @@ bigdecimal.workspace = true
 borsh.workspace = true
 bs58.workspace = true
 bytes = { workspace = true, features = ["serde"] }
-config = { workspace = true, optional = true }
```

Comment:
At the current time `hyperlane-core` has a native `config` module that appears to be replacing this crate.